lib/user/class/applicant.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:myecl/user/class/list_users.dart';
 
 class Applicant extends SimpleUser {
-  late final String email;
+  late final String? email;
```

Comment:
Consider handling the case where email is null (by using .toString() instead of !)